devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -66,6 +67,12 @@ def parse(String description) {
     def resultMap = zigbee.getEvent(description)
     if (resultMap) {
         sendEvent(resultMap)
+        // Temporary fix for the case when Device is OFFLINE and is connected again
+        if (state.lastActivity == null){
+            state.lastActivity = now()
+            sendEvent(name: "deviceWatch-lastActivity", value: state.lastActivity, description: "Last Activity is on ${new Date((long)state.lastActivity)}", displayed: false, isStateChange: true)
```

Comment:
Why are we typecasting lastActivity as long?
Is it not long by default?
As it stores time in epoch format.